faith_presentation/src/commonMain/kotlin/net/thechance/mena/faith/presentation/feature/quran/sur/SurEffect.kt

```diff
@@ -0,0 +1,7 @@
+package net.thechance.mena.faith.presentation.feature.quran.sur
+
+sealed interface SurEffect {
+    data object NavigateBack : SurEffect
+    data object NavigateToBookmark : SurEffect
+    data class NavigateToSurahDetails(val surahId: Int,val surahName: String) : SurEffect
```

Comment:
The formatting is a bit cramped, we can add space between the two parameters 

`val suarhId: Int, val surahName: String`